src/framework/visualization/retrieveIntegrationData.ts

```diff
@@ -0,0 +1,45 @@
+import path from 'path';
+import globby from 'globby';
+
+import { Relationship, Entity } from '../types';
+import { readJsonFile } from '../../fileSystem';
+import { IntegrationData } from './types/IntegrationData';
+import { IntegrationMissingCollectJSON } from './error';
+
+/**
+ * Retrieve integration data from the JSON files from the cache directory
+ */
+export async function retrieveIntegrationData(
```

Comment:
I wonder if this behavior could belong to the `FileSystemGraphObjectStore`? It seems like the kind of storage related operation it should perform and then, should we in the future have other types of graph object storage, the visualizer won't know/care how entities/relationships are loaded.

Reply:
That does make more sense looking at that file, I didn't realize it existed. I'll make that change.

Reply:
Reading @charlieduong94 comments:

> [allow] for people to just write json to the .j1-integration/graph directory by hand for the super simple integrations where devs just want to use j1 as a simple config store. I'd like to still allow for visualizations to work for these developers.

Considering this goal, and having just reviewed `FileSystemGraphObjectStore` code, it seems now it would be best for the visualization code to avoid the `FileSystemGraphObjectStore`. That thing very much expects a certain file system structure, whereas this visualization tool will just walk a tree and figure out what is in the json files.